### PR TITLE
handle request and response for transmittable models

### PR DESCRIPTION
**Pull request by TristanB17**
Implement request and response handling within aca_entities for new IFSV RJ9T payload for REST/JSON within FTI. 

Ticket: https://www.pivotaltracker.com/n/projects/2648255/stories/185701063

### CODE REVIEW

**Review by adamrunner — DISMISSED** [review]

File: aca_entities.gemspec [hunk 1]
```diff
@@ -33,15 +33,16 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
   spec.add_dependency 'deep_merge'
-  spec.add_dependency 'dry-monads', '~> 1.2'
-  spec.add_dependency 'dry-struct', '~> 1.0'
-  spec.add_dependency 'dry-types', '~> 1.0'
```
Comment (saikumar9) [inline aca_entities.gemspec:38]: @TristanB17 @adamrunner Why are we bumping the versions of these gems?

Comment (adamrunner) [reply aca_entities.gemspec:38]: To get it working with the latest versions of the gems that are compatible? We can roll back to older versions if needed, but these updated versions appear to work so they will reduce overhead in doing later version upgrades.

Comment (saikumar9) [reply aca_entities.gemspec:38]: @adamrunner We are trying to understand if these are changes that you must need to make FTI work.

**Review by adamrunner — APPROVED** [review]

**Review by saikumar9 — APPROVED** [review]
lgtm!

**Pull request merged**